docs/governance/politeia.md

```diff
@@ -2,54 +2,56 @@
+## How Politeia Works
 
-Reddit-style **up/down voting** is used for comment sorting only. Up/down votes are not anonymous.
+To vote, stakeholders must time-lock DCR to buy tickets. While tickets are live they can be used to cast a vote for or against each open Politeia proposal. For more information on tickets and their role in Decred's governance, see the [introduction to Decred governance](../governance/introduction-to-decred-governance.md) page.
 
-Ticket-voting on proposals doesn’t happen directly on Politeia, but within [Decrediton](../getting-started/user-guides/decrediton-setup.md) or [politeiavoter CLI](https://github.com/decred/politeia/tree/master/politeiavoter) with dcrwallet
+There is a fee for submitting a proposal (0.1 DCR) to limit the potential for proposal spamming. There is also a fee for registering a Politeia account (0.1 DCR) to limit comment spam and up/down voting sock-puppetry. Fees may be altered if a) they aren't serving as a sufficient spam-deterrent, or b) the price of DCR changes considerably.
 
-**Censorship**  
+### Transparent Censorship
 
 When proposals are submitted, they are checked by Politeia administrators. Proposals that are deemed spam or invalid will be censored.
 
-Politeia is built around the concept of _transparent censorship_, using [dcrtime](https://github.com/decred/dcrtime). Users cannot be silently censored, they can prove that censorship has occurred. When a proposal/comment is submitted, a censorship token is generated. Content creators can use these tokens to demonstrate that their specific proposal/comment was submitted, in a particular form, at a particular point in time.
+Politeia is built around the concept of **transparent censorship**, using [dcrtime](https://github.com/decred/dcrtime). Users cannot be silently censored; they can prove that censorship has occurred. When a proposal/comment is submitted, a censorship token is generated. Content creators can use these tokens to demonstrate that their specific proposal/comment was submitted, in a particular form, at a particular point in time.
 
-**Politeia users have cryptographic identities** (pub/priv key pairs) that can be used to demonstrate censorship of one's submissions. When you register for a Politeia account, a key-pair identity will be generated for you, this is stored in your browser by default.
+Censorship tokens are tied to a **cryptographic identity** (pub/priv key pair) created when you register for a Politeia account. This identity is only used for demonstrating that your submissions have been censored. This key-pair is stored in your browser by default. For now, it is important to **verify your account from the same browser you used to sign up**. 
```

Comment:
> This identity is only used for demonstrating that your submissions have been censored.

This might be better phrased as:

The identity links you to all of your Politeia activity (proposals, comments, comment upvotes/downvotes) and can be used to demonstrate that your submissions have been censored.

Reply:
This inspired me to just rewrite this paragraph to make it more concise:

### Transparent Censorship

When proposals are submitted, they are checked by Politeia administrators. Proposals that are deemed spam or invalid will be censored.

Politeia is built around the concept of **transparent censorship**, using [dcrtime](https://github.com/decred/dcrtime). Users cannot be silently censored; they can prove that censorship has occurred. When a user registers, a **cryptographic identity** (pub/priv key pair) is created. This cryptographic identity is then used to create a "censorship token" for each user submission (proposal, comment, comment upvote/downvote). If a user is censored, these tokens can be used to prove that a specific submission was submitted, the time it was submitted, and the exact form of the submission. This cryptographic identity is stored in the user's browser by default, but can be exported and re-imported at any time. For now, it is important to **verify your account from the same browser you used to sign up**.